### PR TITLE
Citus manager

### PR DESCRIPTION
**Pull request by fapaul**
This PR contains the first version of a helm chart with an integrated membership manager. 

Following points are still missing or/and need some discussion:
* <s>Provisioning does not work very well due to missing readiness checks.</s>
* Port configuration for SQL worker/master access not supported yet.
* Service Account creation is included, might be removed in the future?
* SSL support is pending because this manager version is not released yet.

### CODE REVIEW

**Comment by fapaul** [comment]
@chrboehm As discussed removing the readiness/liveness checks resolves the issue of pg reachability.

File: incubator/citus-postgresql/README.md [hunk 1]
```diff
@@ -102,12 +103,31 @@ helm install incubator/citus-postgresql --name citus-postgresql -f <cluster-name
...
 
+### Manager Deployment Configuration
+
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| `manager.image` | Docker Image for citus-k8s-membership-manager | bakdata/citus-k8s-membership-manager |
+| `manager.imageTag` | Docker Image for citus-k8s-membership-manager | v0.3 |
+| `manager.resources` | Pod resource configuration | - |
+| `manager.namespace` | Namespace in which the citus cluster is supposed to be. | integration-test |
```
Comment (fapaul) [inline incubator/citus-postgresql/README.md]: Change namespace to default

File: incubator/citus-postgresql/templates/pods-list-role-binding.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```
Comment (fapaul) [inline incubator/citus-postgresql/templates/pods-list-role-binding.yaml:2]: Make sa creation optional

File: incubator/citus-postgresql/values.yaml [hunk 3]
```diff
@@ -64,4 +93,6 @@ prometheus:
 ## Secrets
 superuser:
   password: "0987654321"
-  username: "postgres"
\ No newline at end of file
+  username: "postgres"
+  database: "postgres"
```
Comment (fapaul) [inline incubator/citus-postgresql/values.yaml:97]: Add postgres database env variable to citus container

File: incubator/citus-postgresql/README.md [hunk 4]
```diff
@@ -91,6 +91,7 @@ helm install incubator/citus-postgresql --name citus-postgresql -f <cluster-name
 | --------- | ----------- | ------- |
 | `worker.port` | The port on which the Citus worker will be available and serving requests. | `5432` |
```
Comment (fapaul) [inline incubator/citus-postgresql/README.md]: Replace with super user port.

**Review by chrboehm — APPROVED** [review]

**Pull request merged**